app/controllers/admin/books_controller.rb

```diff
@@ -0,0 +1,53 @@
+module Admin
+  class BooksController < AdminController
+    before_action :load_book, only: %i(edit update destroy)
+
```

Comment:
So any type of user can access here, right ?

Reply:
No, because i have BooksController < AdminController. and I had before_action: verify_admin in the AdminController